tests/test_order.py

```diff
@@ -51,10 +53,189 @@ def test_create(self):
             "email": f"test_payer_{random_email_id}@testuser.com"
             }
         }
-        
+
         order_created = self.sdk.order().create(order_object)
         self.assertEqual(order_created["status"], 201)
         self.assertEqual(order_created["response"]["status"], "processed")
 
+        order_get =  self.sdk.order().get(order_created["response"]["id"])
+        self.assertEqual(order_get["status"], 200)
+
+    def test_process_order(self):
+        card_token_id = self.create_test_card()
+        random_email_id = random.randint(100000, 999999)
+        order_object = {
+            "type": "online",
+            "processing_mode": "manual",
+            "external_reference": "ext_ref_1234",
+            "total_amount": "200.00",
+            "transactions": {
+                "payments": [
+                    {
+                        "amount": "200.00",
+                        "payment_method": {
+                            "id": "master",
+                            "type": "credit_card",
+                            "token": card_token_id,
+                            "installments": 1
+                        }
+                    }
+                ]
+            },
+            "payer": {
+                "email": f"test_payer_{random_email_id}@testuser.com"
+            }
+        }
+
+        order_created = self.sdk.order().create(order_object)
+        order_id = order_created["response"]["id"]
+        process_response = self.sdk.order().process(order_id)
+        if process_response.get("status") != 200 or not process_response.get("response"):
+            self.fail(f"Failed to create an order: {process_response}")
```

Comment:
[req] O erro ocorreu no process e não create
```suggestion
        if process_response.get("status") != 200 or not process_response.get("response"):
            self.fail(f"Failed to process an order: {process_response}")
```